File: Phyton class/Assignment/guessingGame.py
import random

number = int(input("Guess a number between 1 and 100:"))
guess = 9;
while (number != guess ):
 if number == guess:
  print ("Congratulation\nYou are correct")
 if number >= guess:
       print("Too high\ntry again!!")
 if number < guess:
         print("Too low\ntry again!!")
count++

File: GuessingGame.java
import java.util.Scanner;
 public class GuessingGame{
   public static void main(String[] args){
    Scanner s = new Scanner(System.in);
     System.out.print("Enter a number from 0-100 :");
     int number = s.nextInt();
     int random = 0;
    if (number == random ){
       System.out.print("Congratulation\nyou are correct");
}else if (number > random){
     System.out.print("Too high\ntry again!!!");
}else if(number < random){
     System.out.print("Too low\ntry again!!!");
}






}






}